transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessageStore.kt

```diff
@@ -25,13 +25,17 @@ internal class MessageStore(
     fun prepareMessage(text: String): OnMessageRequest {
         val messageToSend = pendingMessage.copy(text = text, state = Message.State.Sending).also {
             log.i { "Message prepared to send: $it" }
-            activeConversation.add(0, it)
+            activeConversation.add(it)
             messageListener?.invoke(MessageEvent.MessageInserted(it))
             pendingMessage = Message()
         }
         return OnMessageRequest(
             token = token,
-            message = TextMessage(text, metadata = mapOf("customMessageId" to messageToSend.id), content = messageToSend.getUploadedAttachments()),
+            message = TextMessage(
+                text,
+                metadata = mapOf("customMessageId" to messageToSend.id),
+                content = messageToSend.getUploadedAttachments()
+            ),
```

Comment:
I don't think the trailing comma is needed here

Reply:
@crumpf you are correct. It is not one of recommended use cases for trailing comma. 
https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

Reply:
Thanks for that link! TIL: "The Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it at your discretion for the call site." 🤯